swift/apple/FirezoneNetworkExtension/Adapter.swift

```diff
@@ -139,85 +146,102 @@ class Adapter {
     }
   }
 
-  /// Currently disabled resources
+  /// Internet resource enabled state
   private var internetResourceEnabled: Bool
 
-  /// Cache of internet resource
-  private var internetResource: Resource?
-
-  /// Keep track of resources
+  /// Keep track of resources for UI
   private var resourceListJSON: String?
 
   /// Starting parameters
-  private let apiURL: String
+  private let apiUrl: String
   private let token: Token
-  private let id: String
+  private let deviceId: String
+  private let accountSlug: String
   private let logFilter: String
-  private let connlibLogFolderPath: String
 
-  init(
```

Comment:
Nit: A few more re-orderings and renamings of variable would reduce the diff here!

We are very close :)

Reply:
maybe it's good enough? ;-) after all, I tried to improve them, making them a bit more consistent across Swift and Android!